doc/bc/changes-1.10.md

```diff
@@ -0,0 +1,14 @@
+# Backwards compatibility changes
+
+Changes affecting version compatibility with former or future versions.
+
+## Changes
+- EZP-27633: Implement editing support for Country FieldType
+
+  What changed: `EzSystems\RepositoryForms\FieldType\DataTransformer\CountryValueTransformer` was renamed to `MultipleCountriesValueTransformer`.
```

Comment:
that's a real BC break and this one can be easily avoided. For instance, instead of a creating a new transformer, you could add a `multiple` parameter to the constructor `CountryValueTransformer` (which a default value for BC reason) and then change `transform` and `reverseTransform` to take it into account.

Another option, if you really want to have 2 transformers is to keep a deprecated `CountryValueTransformer` it can either remain as it is now or it can just extend `MultipleCountriesValueTransformer`.